### ISSUE TITLE
ui5-multi-input: [a11y] item cannot be deleted by keyboard, value-help-icon can not be focused and no message for screen readers

### ISSUE THREAD

**Issue by sleric1024** [SAP/ui5-webcomponents#2459]
**Describe the bug**

When using `ui5-multi-input`, we can click the icon by mouse to render a popup. Current issues:

1. The value-help-icon can not be focused by keyboard Tab. As accessibility requirements described: the focus should come to all the interactive elements.
2. When the multi-input control is focused, screen reader should read what control it is and tell the sight disability what he/she could do todo the following thing. For example: when focusing to the value-help-icon, it will tell user _**this will open a popup**_
3.  When several items were selected, currently, users cannot using keyboard such as the arrow key  & backspace to focus to the related item to select and delete.

**Expected behavior**
![2020-11-11_17-10-58 (1)](https://user-images.githubusercontent.com/5724592/98792706-d9c56a80-2441-11eb-8a1d-f8a0495bb2a0.gif)

**Context**
- UI5 Web Components version
- OS/Platform: MacOS
- Browser *(if relevant)*: Chrome
- Other information: {...}

**Affected components** *(if known)*

**Log output / Any errors** in the console
```
{...}
```

**Organization:** *SuccessFactors*

**Comment by fifoosid** [comment]
Hi @sleric1024 

Can you provide an isolated example of how exactly are you using the ```ui5-multi-combobox```. It would be most convenient if you use [CodeSandbox](https://codesandbox.io/s/71r1x5o51q?fontsize=14&module=%2Findex.html).

Also, it would be useful if you provide the exact version of UI5 Web Components that you are using.

**Comment by sleric1024** [comment]
@fifoosid , Thanks for your reply.

- 1 Currently, the item listed in the input can be focused with arrow keys which has the same behavior as UI5 MultiInput did. **But we can not use Del or Backspace key to delete the focused item.**

- 2 In UI5 MultiInput control, it will render with the a11y attributes such as:  `aria-roledescription="Multi Value Input"`, and for value help case: it will contains `aria-describedby="Value help available"  `,  which will be read by screen readers to tell users what the control it is and will be more clear for users to use.

You may refer to this example: (1.1.0-rc-10)
https://codesandbox.io/s/ui5-multi-input-ftipv?file=/index.html

UI5 MultiInput behavior
https://sapui5.hana.ondemand.com/#/entity/sap.m.MultiInput/sample/sap.m.sample.MultiInput
https://sapui5.hana.ondemand.com/#/entity/sap.m.MultiInput/sample/sap.m.sample.MultiInputValueHelp

**Comment by fifoosid** [comment]
FYI @SAP/ui5-webcomponents-topic-rl

**Comment by MapTo0** [comment]
When pressing delete / backspace, `token-delete` event is called and app developer should handle it and remove the token manually. 

https://codesandbox.io/s/ui5-multi-input-forked-5wvrl?file=/src/index.js

**Comment by niyap** [comment]
Hello @sleric1024,

For the first issue reported, please look into the MapTo0 comment above.
For the second one, we will handle it as a feature request because the accessibility specification of the ui5-multi-input component is currently worked on and it is still in progress.

Best Regards,
Niya
